File: Ciclo2/Unidad1/EjerciciosSesión3/src/Calculadora.java
import java.util.Scanner;

public class Calculadora {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int num_1, num_2, option;
        float resultado = 0;
        System.out.println("Operaciones Disponibles para realizar: " +
                "\n1)Suma\n2)Resta\n3)Multiplicación\n4)División \n5)Residuo \n6) Salir.");
        System.out.print("Elija una opción: ");
        option = in.nextInt();
        if(option == 6){
            System.out.println("Hasta Pronto");
        }
        else if (option >=0 && option <=5){
            System.out.print("Ingrese el primer Valor: ");
            num_1 = in.nextInt();
            System.out.print("Ingrese el segundo Valor: ");
            num_2 = in.nextInt();
            switch (option){
                case 1:
                    resultado = num_1 + num_2;
                    break;
                case 2:
                    resultado = num_1 - num_2;
                    break;
                case 3:
                    resultado = num_1 * num_2;
                    break;
                case 4:
                    if(num_2!=0){
                        resultado = num_1 / num_2;
                    }else{
                        System.out.println("Error al realizar la operación");
                        System.exit(0);
                    }
                    break;
                case 5:
                    resultado = num_1 % num_2;
                    break;
                default:
                    System.out.println("Entrada Incorrecta");
                    break;
            }
            if (option <= 5){
                System.out.println("El resultado de la operación es: "+resultado);
            }
        } else{
            System.out.println("Entrada Incorrecta");
        }

    }
}

File: Ciclo1/Unidad2/actividad1/calculadora.py
menu = """
Bienvenido A nuestra calculadora 馃柀
1 - Suma
2 - Resta
3 - Multiplicacion
4 - Divisi贸n
5 - Potencia
Elige la operaci贸n a realizar: """

opcion = int(input(menu))

if opcion == 5: #Si es potencia entra a este condicional
    num = int(input('Ingrese el numero a potenciar: '))
    resultado = num ** 2
    print(resultado)
else: # Si no valida el resto de operaciones
    num_1 = int(input('Ingrese el primer numero: '))
    num_2 = int(input('Ingrese el segundo numero: '))
    if opcion == 1: # Valida la resta
        suma = num_1 + num_2
        print(f'el resultado de la suma es = {suma}')
    elif opcion == 2: # Valida la resta
        resta = num_1 - num_2
        print(f'el resultado de la resta es = {resta}')
    elif opcion == 3: #Valida la Multiplicaci贸n
        multi = num_1 * num_2
        print(f'el resultado de la multiplicacion es = {multi}')
    elif opcion == 4: # Valida la divisi贸n
        if num_2 != 0: #Valida que el denominador sea diferente de 0
            div = num_1 / num_2
            print(f'el resultado de la Divisi贸n es = {div}')
        else:
            print('La division no es valida')
    else:
        print('Ingrese una operacion valida')
